pkg/crc/config/callbacks.go

```diff
@@ -33,6 +33,11 @@ func RequiresCRCSetup(key string, _ interface{}) string {
 		"Please run 'crc setup' for this configuration to take effect.", key)
 }
 
+func RequiresCleanupAndSetupMsg(key string, _ interface{}) string {
+	return fmt.Sprintf("Changes to configuration property '%s' are only applied during 'crc setup'.\n"+
+		"Please run 'crc cleanup' followed by 'crc setup' for this configuration to take effect.", key)
+}
```

Comment:
Follow-up issue to make this more dynamic would be useful to have.

Reply:
https://github.com/code-ready/crc/issues/3367